Alignment/LaserAlignmentSimulation/src/LaserOpticalPhysicsList.cc

```diff
@@ -96,7 +96,7 @@ void LaserOpticalPhysicsList::ConstructProcess() {
   pManager->AddDiscreteProcess(theWLSProcess);
 
   theScintProcess->SetScintillationYieldFactor(1.);
-  theScintProcess->SetScintillationExcitationRatio(0.0);
+  //theScintProcess->SetScintillationExcitationRatio(0.0);
```

Comment:
Hi, would there be some reason to keep this line commented instead of just removing it?

Reply:
no real reason, this interface is obsolete.